web/admin/src/App.js

```diff
@@ -21,23 +21,25 @@ export default class App extends Component {
   componentDidMount() {
     fbc.signinAdmin()
     .then(() => {
-      client.getUsers().then(attendees => this.setState({attendees: attendees.sort(sortPlayers)}))
+      client.getUsers().then(attendees => {this.setState({attendees: attendees.sort(sortPlayers)}) 
 
-      const usersRef = fbc.database.public.usersRef()
-      usersRef.on('child_added', data => {
-        this.setState(state => ({players: [...state.players, {...data.val(), id: data.key}].sort(sortPlayers)}))
-      })
-      usersRef.on('child_removed', data => {
-        this.setState(state => ({players: state.players.filter(p => p.id !== data.key)}))
-      })
-
-      fbc.database.public.adminRef('targets').on('value', data => {
-        this.setState({isGameInProgress: !!data.val()})
-      })
-
-      fbc.database.private.adminableUsersRef().on('value', data => {
-        const users = data.val() || {}
-        this.setState({admins: Object.keys(users).filter(id => users[id].adminToken)})
+        const usersRef = fbc.database.public.usersRef()
+        usersRef.on('child_added', data => {
+          var player = attendees.find(newUser => newUser.id === data.val().id)
```

Comment:
`newUser` is a misnomer, because in attendees.find(), it will be every user until it finds the one that matches `data.val().id` (not just the new one being added).  I would name it either `attendee` or simply `a` (which is still readable since it is an extremely short `=>` function)